test/fixtures/service_fixtures.js

```diff
@@ -281,7 +298,10 @@ module.exports = {
       },
       redirect_to_service_immediately_on_terminal_state: false,
       collect_billing_address: false,
-      current_go_live_stage: 'NOT_STARTED'
+      current_go_live_stage: 'NOT_STARTED',
+      merchant_details: {
+        name: opts.value || ''
+      }
```

Comment:
I don't think we want to do this, as we can have a service response with no merchant details if there are none defined.
If you pass in any merchant details in the opts currently, it will build with the defaults defined in buildMerchantDetailsWithDefaults.

Reply:
Just remove this change and I think everything will be fine